clients/cloud/python/README.md

```diff
@@ -17,6 +17,24 @@ sasl.username=<api-key-id>
 sasl.password=<secret-access-key>
 ```
 
+## Configure SSL trust store
+
+Depending on your operating system or Linux distro you may need to take extra steps to set up the SSL CA root certificates.
```

Comment:
It would be helpful to enumerate which OS/distros need to take these extra steps and which do not.